r_standards.md

```diff
@@ -0,0 +1,50 @@
+# R Style Guidelines
+The ACF Data Surge team's R Style Guidelines are intended to make our R code consistent and easy to read across projects. Our guidelines align with the [The tidyverse style guide](https://style.tidyverse.org/) and we adopt the [Google R Style Guide](https://google.github.io/styleguide/Rguide.html) approach of listing the ways that our guidelines differ from the tidyverse guidelines. 
+
+Our guidelines are unique in that we include a section at the end detailing preferred packages and package management standards.
+
+## Functions
+
+### Returning Values
+
+- Use return() explicitly for returning values from a function
+```
+my_mean <- function(x) {
+    x_mean <- mean(x)
+    return(x_mean)
+}
+```
+
+## Pipes
+
+### Assignment
+- Avoid right-handed assignment
+```
+iris %>%
+    arrange(value) ->
+    iris
+```
+
+## Packages
+
+### Package Management
+We will use [renv](https://rstudio.github.io/renv/articles/renv.html) to manage packages for our projects/code written in R. When beginning a new project, create a dedicated R project. This R project, and attendant renv related files, should be tracked on GitHub so that all contributors are working in the same development environment. See the [renv](https://rstudio.github.io/renv/articles/renv.html) documentation for a list of files that should be tracked in Git. Be sure to also include the .Rproj file for the project.
```

Comment:
In second URL, please include "documentation" into the hyperlink text to make it more accessible.